posts/Dmitry/2017-02-11-EclipseToolsForHibernateSearchRelease20.adoc

```diff
@@ -0,0 +1,39 @@
+= Eclipse tools for Hibernate Search
+Dmitry  Bocharov
+:awestruct-tags: [ "Hibernate Search", "JBoss Tools" ]
+:awestruct-layout: blog-post
+---
+I'm glad to announce the second release of the https://marketplace.eclipse.org/content/hibernate-search-plugin[Eclipse plugin for Hibernate Search].
+In this post I'm describing the changes and new features of the release. http://in.relation.to/2016/01/22/EclipseToolsForHibernateSearch[Here] you can find the first release blog post.
+
+
+== Index Toolkit
+
+The major change of the release is that all instruments, except "Index Rebuild", are now grouped in the Index Toolkit view. To open it, right-click on a configuration in the Hibernate Configurations view and choose "Open Index Toolkit".
```

Comment:
"instruments" ? Is that an Eclipse technical term? I understand what you mean but it doesn't seem fitting.